history-of-the-quakers-v2/original/08-book-3-chapter-14.adoc

```diff
@@ -25,26 +31,29 @@ pestilence, or any symptom of divine displeasure, were calamities of equal magni
 with the existence of non-conformity, they proceeded to enact a fresh penal law,
 commonly known by the name of The Oxford five-mile act,
 which received the royal assent October the 31st, 1665.
-It was entitled, An aft to restrain non-conformists from inhabiting corporations;
-the preamble to which sets forth,
+It was entitled, An Act to Restrain Non-conformists from Inhabiting Corporations;
```

Comment:
This is sort of random (meaning we don't really have a rule about how to handle these), but since you're editing this section, can you italicize this act title? Just normal italics, not a [.book-title].

Also, italicize the Oxford five-mile act above as well 👍